### DISCUSSION TITLE
Possible to bring the Apple Porting kit DX11/12 libraries into Wineskin?

### DISCUSSION THREAD

**Discussion by palmerj** [The-Wineskin-Project/WineskinServer#409] [General]
As per https://www.applegamingwiki.com/wiki/Game_Porting_Toolkit#Game_compatibility_list

**Comment by Gcenx** [comment]
There’s not much point linking AppleGamingWiki for this when Apples own documentation is already good, it also makes many incorrect assumptions that simply won’t apply to Wineskin.

You may have noticed WineCX22.1.1-2 this already works on macOS Sonoma beta but additional changes are required to have Apples dxmetal translation working correctly.

**Reply by palmerj** [reply]
Great thank you, and sorry for posting to the wrong docs :-)

**Reply by palmerj** [reply]
Just tested out the new WineCX22.1.1-2 (WineCX22.1.1-1 is the same too), and the current game I'm playing doesn't work). I will raise another issue for that.
